template.go

```diff
@@ -47,11 +47,12 @@ type template struct {
 	RoutingKey      string `yaml:"routingKey" yaml-aliases:"routingkey,routing-key,rk"`
 	RoutingDelegate string `yaml:"routingDelegate" yaml-aliases:"routingdelegate,routing-delegate,rd"`
 
-	Headers map[string]string           `yaml:"headers"`
-	Baggage map[string]string           `yaml:"baggage"`
-	Jaeger  bool                        `yaml:"jaeger"`
-	Request map[interface{}]interface{} `yaml:"request"`
-	Timeout time.Duration               `yaml:"timeout"`
+	Headers  map[string]string             `yaml:"headers"`
+	Baggage  map[string]string             `yaml:"baggage"`
+	Jaeger   bool                          `yaml:"jaeger"`
+	Request  map[interface{}]interface{}   `yaml:"request"`
+	Requests []map[interface{}]interface{} `yaml:"requests"`
```

Comment:
what happens if both are set? should we return an error (I think we ignore `Request` silently right now)

Reply:
I missed adding this check earlier, now it returns error if both fields are set